test/functional/buildAndPackage/src/net/adoptopenjdk/test/VendorPropertiesTest.java

```diff
@@ -49,8 +61,17 @@ public VendorPropertiesTest() {
                 .orElseThrow(() -> new AssertionError("No checks found for vendor: " + vendor));
     }
 
+    /**
+     * Verifies that the vendor name is displayed within
+     * the java -version output, where applicable.
+     */
     @Test
     public void javaVersionPrintsVendor() {
+        // Skip test on JDK8 for non-Hotspot JDKs.
+        if (!jdkVersion.isNewerOrEqual(9) && !jdkVersion.usesVM(VM.HOTSPOT)) {
```

Comment:
I think the intended condition was:
```
    !jdkVersion.isNewerOrEqual(9) || !jdkVersion.usesVM(VM.HOTSPOT)
```

Reply:
You'd think so, but no. The vendor name is only absent from builds which are both OpenJ9 *and* JDK8. 

Hotspot JDK8, and Hotspot+OpenJ9 JDK9+ builds all have the vendor name in the -version output because we added it to our fork of OpenJDK8, but OpenJ9's fork doesn't have that addition. We asked about that, and they said they only want it if our change gets proposed+accepted upstream (if anyone ever asks for it).

Reply:
I make it !hotspot (rather than openj9-specific) because other people's forks of the jdk8 source won't have that change either, unless they add it. Future-proofing.

Reply:
If it's only missing from OpenJ9 and JDK8, then I suggest a clearer (and more accurate) condition would be
```
    jdkVersion.usesVM(VM.OPEN9) && !jdkVersion.isNewerOrEqual(9)
```

Reply:
I don't understand how that would be more accurate. The only known instance of a jdk8 fork that adds the vendor name to the -version output is Adopt's fork of jdk8 for Hotspot. I don't think we should expect other forks of the jdk8 repo to include a change which may be specific to ours.

That said, as others may have jdk8 hotspot forks which are not Adopt's fork, we could try for absolute accuracy by skipping the test for all jdk8 builds that are either (A) non-Hotspot, or (B) non-Adopt.

E.g.
```
        // Skip test on JDK8 where the VM is not Hotspot, or if the JDK8 build vendor is not Adoptium.
        boolean buildVendorIsAdoptium = System.getProperty("java.vendor.url").equals("https://adoptium.net/");
        if (!jdkVersion.isNewerOrEqual(9) && (!jdkVersion.usesVM(VM.HOTSPOT) || !buildVendorIsAdoptium)) {
            return;
        }
```